reana_client/cli/quotas.py

```diff
@@ -52,11 +55,13 @@ def quota_group():
     type=click.Choice(["limit", "usage"], case_sensitive=False),
     help="Specify quota report type. e.g. limit, usage.",
 )
-@quota_group.command("quota-show")
-@click.pass_context
+@human_readable_option
```

Comment:
Minor: WDYT about making this human-readable by default? Most cases the user need meaningful numbers here

`296 used`
vs
`46s out of 40s used (117%)`

Anoher approach could be displaying the canonical units when no `--human-readable` passed.

Reply:
This is something I thought while implementing, but I saw it a bit controversial.. however, now thinking it twice, this output is not machine-readable anyways... so I don't see why not doing it `human_readable` by default. I am changing it and if we are not happy with it we can very easily revert it.